exercises/little-schemer/canonical-data.json

```diff
@@ -0,0 +1,215 @@
+{
+  "exercise": "little-schemer",
+  "version": "1.0.0",
+  "comments": [
+    "Tests are ordered to allow sequential build up of primitives",
+    "for use in writing the final recursive function lat?."
+  ],
+  "cases": [
+    {
+      "description": "Test null? of a list",
+      "cases": [
+        {
+          "description": "null of non-empty list",
+          "property": "boolean",
+          "input": {
+            "function": "null?",
```

Comment:
Those from languages where `?` is not a valid identifier character will probably object to this.